.github/workflows/ci.yml

```diff
@@ -111,6 +111,7 @@ jobs:
                 . ./ci-support-v0
                 build_py_project_in_conda_env
                 ( test_py_project )
+                export LOOPY_ABORT_ON_CACHE_MISS=1
                 ( test_py_project )
```

Comment:
@inducer do you remember what the purpose of this `pytest_twice` check was/is? Is it just about seeing the time difference between the first pytest run and the second?

Reply:
No, it's supposed to test that things still work with data pulled out of a cache, since the first round of tests runs on a cold cache by default.

Reply:
Is it expected that running pytest like this will lead to cache misses on the second run (demonstrated by the CI in this PR)? I've tried to go back to earlier loopy versions, and it seems like these cache misses are not caused by recent changes.

Reply:
Huh, I would not expect there to be cache misses on the second go.

Reply:
I suspect code like this (from `test_c_execution.py`) is responsible for the cache misses:

```python
def __get_kernel(order="C"):
    indices = ["i", "j", "k"]
    sizes = tuple(np.random.randint(1, 11, size=len(indices)))
    # ...
```

Would you prefer me to make these sizes constant across runs, or should we just ignore the cache miss?

Reply:
Setting a seed (and converting to the new-style numpy RNG interface) seems like the way to go.

Reply:
Considering the multitude of issues encountered, it is probably not realistic to enable this in CI for the near future. I'll disable the export for now.